lib/src/base-styles/_nx-divider.scss

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-divider {
+  background-color: var(--nx-swatch-indigo-90);
+  border-radius: 1px;
+  border-style: none;
+  margin: 24px 0;
+  min-height: 1px;
```

Comment:
Is there a reason this isn't just `height`?

Reply:
> Is there a reason this isn't just `height`?

This was a 'to-be-removed' remains of me messing around with both horizontal and vertical borders.
Check 1c8d806169e9f3fcf770a7d8dc42b9e9ab95436c